code/obj.dm

```diff
@@ -23,6 +23,34 @@
 	var/_health = 100
 	var/_max_health = 100
 
+	New()
+		. = ..()
+		SPAWN_DBG(0) // needs a chance to set dir apparently
```

Comment:
apparently? 😳

Reply:
No you're right. It's because I didn't remember to handle `set_dir` and that confused my testing using object spawn